Content.Server/Ame/EntitySystems/AmeControllerSystem.cs

```diff
@@ -106,18 +110,37 @@ public void UpdateUi(EntityUid uid, AmeControllerComponent? controller = null)
 
         var state = GetUiState(uid, controller);
         _userInterfaceSystem.SetUiState(bui, state);
+
+        controller.NextUIUpdate = _gameTiming.CurTime + controller.UpdateUIPeriod;
     }
 
     private AmeControllerBoundUserInterfaceState GetUiState(EntityUid uid, AmeControllerComponent controller)
     {
+
         var powered = !TryComp<ApcPowerReceiverComponent>(uid, out var powerSource) || powerSource.Powered;
-        var coreCount = TryGetAMENodeGroup(uid, out var group) ? group.CoreCount : 0;
+        var coreCount = 0;
+        // how much power is currently drawn from the AME, in kW
+        float currentPowerSupply = 0;
```

Comment:
Move the declaration closer to the first use

Reply:
ok